catalog/samples/hello.py

```diff
@@ -0,0 +1,9 @@
+import sys
+def main(dict):
+    if 'user' in dict:
+        print("hello, " + dict['user'] + "!")
+        return {"user":dict['user']}
+    else:
+        print("hello world!")
+        return {"user":"world"}
```

Comment:
the swift and java examples return { greeting: "Hello stranger!" } in the else branch.